core/env/each_value_spec.rb

```diff
@@ -10,7 +10,7 @@
       ENV.clear
       ENV["1"] = "3"
       ENV["2"] = "4"
-      ENV.each_value { |v| e << v }
+        ENV.each_value { |v| e << v }.should equal(ENV)
```

Comment:
I don't understand this.  There's no extra space in line 13 over at https://github.com/BurdetteLamar/spec/blob/master/core/env/each_value_spec.rb.

Reply:
It seems fine now, maybe it was a tab or a bug of GitHub.